go.mod

```diff
@@ -102,7 +102,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/denis-tingajkin/go-header v0.3.1 // indirect
 	github.com/docker/cli v20.10.7+incompatible // indirect
-	github.com/docker/distribution v2.7.1+incompatible // indirect
+	github.com/docker/distribution v2.8.0+incompatible // indirect
```

Comment:
I imagine because of the `replace` directive this is effectively a no-op. It looks like the entire `replace` block can be removed now though since Helm doesn't need it anymore. Then this change would actually bump the version we build with.

More context: https://github.com/openservicemesh/osm/pull/1306#pullrequestreview-460228804

Reply:
We should be able to remove that `replace` directive then right?

Reply:
Yes

Reply:
Sorry just reread your comment. Ok, sounds good. I'll remove the `replace` block and retest.